aws/common_columns.go

```diff
@@ -127,8 +127,8 @@ func getCommonColumnsUncached(ctx context.Context, d *plugin.QueryData, h *plugi
 		region = "global"
 	}
 
-	// Trace logging to debug cache and execution flows
-	plugin.Logger(ctx).Trace("getCommonColumnsUncached", "status", "starting", "connection_name", d.Connection.Name, "region", region)
+	// Debug logging to debug cache and execution flows
+	plugin.Logger(ctx).Debug("getCommonColumnsUncached", "status", "starting", "connection_name", d.Connection.Name, "region", region)
```

Comment:
How often does this log appear when a query runs successfully? I'm worried about how noisy it could be when queries are working successfully. When a query is run unsuccessfully, it could make it hard to finding the relevant log messages.

Reply:
I think it would be better to remove the log statements in each step. How it would be if we will have a single debug statement at the end of the successful execution? and the error statements while handling the error?

I believe that eliminating log statements at every step would be better. What are your thoughts on having a single debug statement upon successful completion and error statements specifically for errors wherever we handle the error?

Reply:
I think leaving them in at their current points is OK, it's helpful to walk through with them in order. I think they should remain at `Debug` level too